### PR TITLE
feat: add raw transaction builder

### PR DESCRIPTION
**Pull request by m-cote**
Fixes #SB-241

## Description of the changes

- adds rawTransactionBuilder to create a raw transaction without sending it to the network

### CODE REVIEW

**Review by Kozer4 — APPROVED** [review]

File: src/bridge/evm/index.ts [hunk 1]
```diff
@@ -105,7 +165,7 @@ export class EvmBridge extends ApprovalBridge {
     return new this.web3.eth.Contract(abiItem, contractAddress) as any;
```
Comment (YaroslavNekryach) [inline src/bridge/evm/index.ts:165]: This is the only place where you use web3. For this case you can create empty web3 instance by yourself, so user doesn't need to pass it in this class

Comment (YaroslavNekryach) [reply src/bridge/evm/index.ts:165]: I wanted to say, that for building raw transaction you do not need a connection

Comment (m-cote) [reply src/bridge/evm/index.ts:165]: It is the case for web3. But I couldn't manage to make it work for TronWeb. TronWeb still makes some requests to a node when creating a transaction.
Are you suggesting to make a parameter `provider` optional only when Web3 is required and mandatory when it has to be TronWeb?

Comment (YaroslavNekryach) [reply src/bridge/evm/index.ts:165]: Maybe, need to think

File: src/bridge/trx/index.ts [hunk 2]
```diff
@@ -7,9 +7,11 @@
 import { TronWeb } from "tronweb-typings";
```
Comment (YaroslavNekryach) [inline src/bridge/trx/index.ts:7]: Don't use old typings from an unknown package, it confuses more than helps

File: src/bridge/evm/index.ts [hunk 3]
```diff
@@ -75,6 +76,46 @@ export class EvmBridge extends ApprovalBridge {
...
+      messenger
+    );
+
+    return new Promise((resolve) =>
+      resolve({
+        from: fromAccountAddress,
+        to: contractAddress,
+        value: fee,
+        data: swapAndBridgeMethod.encodeABI(),
+        maxFeePerGas: 100000000000,
```
[the remark this answers is not in the document]
Comment (m-cote) [reply src/bridge/evm/index.ts]: I agree. I removed these fields.

File: src/bridge/trx/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (m-cote) [reply src/bridge/trx/index.ts]: I disabled in .eslintrc some eslint checks that fail with using TronWeb

Comment (m-cote) [reply src/bridge/trx/index.ts]: I agree. I removed `tronweb-typings`

**Pull request merged**